simSPI/linear_simulator/configs/betagal_distribution.cfg

```diff
@@ -0,0 +1,71 @@
+[general]
+    name = betagal_distribution
+            #str: name of the dataset folder
+    usegpu= False
+            #bool: (default: False) option to use GPU TODO: gpu option to be added
+    datasetSize= 100
+            #int: (default: 410) size of the dataset
+    chunks=10
+            #int: (default: 10) size of the chunk per iteration
+
+[paths]
+    input_volume_path = simSPI/linear_simulator/configs/betagal.mrc
+            #str: path to the mrcfile
+    output_path= simSPI/linear_simulator/
+            #str: path of the output directory in which the 'dataset_name' folder is created
+    input_starfile_path=
+            #str: path to the input starfile from which parameters are read.
+            #     If left blank then the generator creates parameters from the specified distributions
+
+[projector]
+    volumedomain= space
+            # str: (default: space, options: fourier, space) type of projector
+    sidelen = 64
+            # int: (default: 64) length of the volume in pixels
+    angle_distribution= uniform
+            # str: (default: uniform) distribution from which rotation angles are sampled over SO3
+    relion_invert_hand = False
+            # bool: (default: False) option to invert the hand of the volume
+[ctf]
+    ctf = True
+            # bool: (default: True) option to enable ctf convolution
+    changectfs = True
+            # bool: (default: True) option to change ctf per image
+    ctf_size = 64
+            # int: (default: str) size of the ctf in pixels. Currently, in the code this is forecefully made equal to sidelen
+    pixel_size= 1
+            # int: (default: 1)   real size of the pixel of the detector in Angstrom (A)
+    kV= 300
+            # float: (default: 300) voltage in kV
+    valueNyquist= 0.2
+            # float: (default: 0.2) value of the envelope at the nyquist frequency
+    bfactor=0
+            # float: (default: 0) value of the envelope B-factor
+
+    cs=2.7
+            # float: (default :2.7) spherical abberation in mm
```

Comment:
cs_mm

Reply:
I prefer cs notation. Its more readable. I added that its in mm in the docstring of CTF. Done in #27.